developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md

```diff
@@ -0,0 +1,155 @@
+---
+title: text2vec-jinaai
+sidebar_position: 13
+# tags: ['text2vec', 'text2vec-jinaai', 'jinaai']
+---
+
+
+## Overview
+
+Weaviate uses the `text2vec-jinaai` module to obtain vectors.
+- [JinaAI Embeddings](https://jina.ai/embeddings/)
+
+Key notes:
+
+- As it uses a third-party API, you will need an API key.
+- Enabling this module will enable the [`nearText` search operator](/developers/weaviate/api/graphql/search-operators.md#neartext).
```

Comment:
Consider:
To use the [`nearText` search operator], enable the `text2vec-jinaai` model.